website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -19,6 +22,12 @@ When you set this configuration, dbt sends a helpful error message for any user
 
 If this configuration is not specified, no version check will occur.
 
+:::info Keep on latest version <Lifecycle status='beta' />
+
+<Snippet path="_config-dbt-version-check" />
+
+:::
+
 ### YAML quoting
 
 This configuration needs to be interpolated by the YAML parser as a string. As such, you should quote the value of the configuration, taking care to avoid whitespace. For example:
```

Comment:
The note below (at line 75) is not closed (no closing `:::`) so it's making the entire section look like a note. 

> :::info Not recommended

Reply:
great catch! ty!